container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/consts/consts.go

```diff
@@ -33,6 +33,7 @@ const (
 	HttpApplicationProtocol                             = "http"
 
 	GitHubAuthStorageDirPath = "/kurtosis-data/github-auth/"
+	GithubAuthStorageToken   = "token.txt"
```

Comment:
Out of scope of this PR and not related to your change but just calling out - but at this point LOTS of the codebase relies on  the hardcoded`/kurtosis-data` directory path. I believe it's originally set as env var to the APIC, but somehow the directory path needs to be refactored and propagated to the elements that need it.